src/library/scala/collection/immutable/Queue.scala

```diff
@@ -60,7 +60,8 @@ class Queue[+A] protected(protected val in: List[A], protected val out: List[A])
     if (n < len) out.apply(n)
     else {
       val m = n - len
-      if (m < in.length) in.reverse.apply(m)
+      val l = in.length
+      if (m < l) in.apply(l - m - 1)
```

Comment:
Some might object to readability of `l - m - 1`.  Math is hard enough without lower case ell. Or e11.

I don't have a good punning name. They already took m and n. Maybe `nel` for `len` in reverse.